components/search_bar/search_bar.tsx

```diff
@@ -112,7 +111,7 @@ const SearchBar: React.FunctionComponent<Props> = (props: Props): JSX.Element =>
         >
             <form
                 role='application'
-                className={classNames(['search__form', {'search__form--focused': isFocused, 'search__form--global': props.globalHeaderEnabled}])}
+                className={classNames(['search__form', 'search__form--global', {'search__form--focused': isFocused}])}
```

Comment:
Should the styles from `search__form--global` be rolled into `search__form` now since both classes are always applied at the same time?